### ISSUE TITLE
Fedora 35 beta + Python error

### ISSUE THREAD

**Issue by Anonymo** [silenc3r/fedora-better-fonts#36]
sudo dnf copr enable dawid/better_fonts -y
/usr/lib/python3.10/site-packages/dnf-plugins/copr.py:433: DeprecationWarning: distro.linux_distribution() is deprecated. It should only be used as a compatibility shim with Python's platform.linux_distribution(). Please use distro.id(), distro.version() and distro.name() instead.
  dist = linux_distribution()

**Comment by bkdwt** [comment]
When I try to install on Fedora 35, I get this error:

```
Error: 
 Problema: solicitações conflitantes
  - nothing provides archivo-black-fonts needed by fontconfig-font-replacements-0.6-2.fc35.noarch
(tente adicionar '--skip-broken' para pular pacotes desinstaláveis)
```

**Comment by jonprocter** [comment]
> When I try to install on Fedora 35, I get this error:
> 
> ```
> Error: 
>  Problema: solicitações conflitantes
>   - nothing provides archivo-black-fonts needed by fontconfig-font-replacements-0.6-2.fc35.noarch
> (tente adicionar '--skip-broken' para pular pacotes desinstaláveis)
> ```

Yep me too - just did a clean install of the latest F35 beta and having the same problem as you.

**Comment by sandrocarval** [comment]
Same on a brand new (f35) RC1.2 installation (which will be the final/production build).

```
nothing provides archivo-black-fonts needed by fontconfig-font-replacements-0.6-2.fc35.noarch
```

**Comment by sn99** [comment]
@Anonymo rename to `Missing archivo-black-fonts on Fedora 35`

**Comment by MayCXC** [comment]
Looking [here](https://copr.fedorainfracloud.org/coprs/dawid/better_fonts/build/2077386/) I see that the f35 repo has not been built yet, not sure how we can do that ourselves...

**Comment by maxreuben** [comment]
Currently on F35, there seems to be a [workaround](https://github.com/silenc3r/fedora-better-fonts/issues/37#issuecomment-973183507). I can confirm that this works on Fedora 35.
